### PR TITLE
UPAKFromUPKV2

### PR DESCRIPTION
**Pull request by oconnor663**
r? @maxtaco

### CODE REVIEW

**Comment by oconnor663** [comment]
Am I right that the first instantiation of a UPKV2AI will be in https://keybase.atlassian.net/browse/CORE-5319?

File: go/protocol/keybase1/extras.go [hunk 1]
```diff
@@ -1154,3 +1154,96 @@ func (m *MaskB64) MarshalJSON() ([]byte, error) {
...
+			PGPKeyCount:       len(pgpKeysV1),
+			Uvv:               uV2.Current.Uvv,
+			PerUserKeys:       uV2.Current.PerUserKeys,
+		},
+		PGPKeys:      pgpKeysV1,
+		RemoteTracks: uV2.Current.RemoteTracks,
+	}
+
+	// Return a defensive copy.
+	return upak.DeepCopy()
```
Comment (maxtaco) [inline go/protocol/keybase1/extras.go]: This might evade some of our optimizations.  I'd remove it for now.

**Comment by oconnor663** [comment]
Known flake according to @jzila:

```
19:41:40 19:41:40 [8253] Running shell script
19:41:41 19:41:41 + rm -rf /Volumes/untitled/kbfs
19:41:41 19:41:41 rm: /Volumes/untitled/kbfs/fusetest139438538: Device not configured
19:41:41 19:41:41 rm: /Volumes/untitled/kbfs/fusetest947318871: Device not configured
19:41:41 19:41:41 rm: /Volumes/untitled/kbfs: Directory not empty
```

https://ci.keyba.se/job/kbfs/job/master/8253/execution/node/271/log/

**Comment by jzila** [comment]
@oconnor663: FYI you linked the wrong flake, the one you included is ignored.
Actual flake:
https://ci.keyba.se/job/kbfs-server/job/master/12946/execution/node/173/log/

**Comment by oconnor663** [comment]
Woops!

**Pull request merged**